std/algorithm/comparison.d

```diff
@@ -902,7 +913,8 @@ template equal(alias pred = "a == b")
         for element, according to binary predicate `pred`.
     +/
     bool equal(Range1, Range2)(Range1 r1, Range2 r2)
-    if (isInputRange!Range1 && isInputRange!Range2 &&
+    if (!useCodePoint!(Range1, Range2) &&
```

Comment:
Now we have an undocumented artifact `useCodePoint` used in the interface.

Reply:
I can spell out the `useCodePoint` thing, or make it more public. I have a feeling it's going to be needed elsewhere anyway.